application/routes/user.py

```
@@ -80,7 +89,9 @@
     try:
         username = request.json['username']
         profiles.send_activation_email(username)
+        logger.info("User: Activation email resent to user: %s", username)
     except KeyError as e:
+        logger.error("User: Missing required field during activation email resend: %s", e)
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)